CondTools/DT/plugins/DTKeyedConfigDBDump.cc

```diff
@@ -21,20 +26,36 @@
 #include "FWCore/Framework/interface/ESHandle.h"
 #include "FWCore/Framework/interface/MakerMacros.h"
 
-//---------------
-// C++ Headers --
-//---------------
-#include <iostream>
-#include <memory>
+//-----------------------
+// This Class' Header --
+//-----------------------
+class DTKeyedConfigDBDump : public edm::EDAnalyzer {
```

Comment:
Please migrate to a non-legacy and thread aware EDAnalyzer
(This could be done also for the analyzers in the \test areas, but since they are only supposed to be run privately, this is not strictly required)

Reply:
Updated 2 modules in the plugins directory.